scripts/build.sh

```diff
@@ -11,7 +11,7 @@ GN_ARGS_BASE="
   v8_use_snapshot=true
   v8_use_external_startup_data=false
   icu_use_data_file=false
-  v8_enable_lite_mode=true
+  v8_enable_lite_mode=${DISABLE_JIT}
```

Comment:
Just a nit.
Could you setup `v8_enable_lite_mode` like what NO_INTL did?
I.e.
Removing this line and add
```sh
if [[ ${DISABLE_JIT} -ne "false" ]]; then
  GN_ARGS_BASE="${GN_ARGS_BASE} v8_enable_i18n_support=false"
fi
```

Otherwise, for DISABLE_JIT mode, the generated GN config will be:
`v8_enable_lite_mode=` Explicit is better than implicit.

Reply:
i've made the change but i'm not sure its correct what you requested (in terms of true/false)
what is the default if you dont specify v8_enable_lite_mode?

Reply:
Yes, the change is exactly what I talked about.
The default value should be JIT enabled.

I am going to merge this PR and publish JIT versions once builds ready.
Thank you so much for your fantastic work.